opus/application/static_media/js/browse.js

```diff
@@ -174,7 +174,7 @@ var o_browse = {
                     o_cart.toggleInCart(fromOpusId, opusId);
                 }
             } else {
-                o_browse.showModal(opusId);
+                o_browse.showGalleryViewModal(opusId);
```

Comment:
Why is this called GalleryViewModal? Doesn't it also work in table view? Perhaps call it "BrowseModal" (except it works in cart...) or "MetadataDetailModal"?